src/sentry/api/endpoints/organization_events_trends.py

```diff
@@ -83,11 +103,16 @@ def data_fn(offset, limit):
                 + [
                     trend_column["format"].format(*columns, start=start, end=middle, index="1"),
                     trend_column["format"].format(*columns, start=middle, end=end, index="2"),
-                    percentage_column["format"].format(alias=trend_column["alias"]),
                     "minus({alias}2,{alias}1)".format(alias=trend_column["alias"]),
                     count_column["format"].format(start=start, end=middle, index="1"),
                     count_column["format"].format(start=middle, end=end, index="2"),
                     percentage_column["format"].format(alias=count_column["alias"]),
+                    variance_column["format"].format(start=start, end=middle, index="1"),
+                    variance_column["format"].format(start=middle, end=end, index="2"),
+                    avg_column["format"].format(start=start, end=middle, index="1"),
+                    avg_column["format"].format(start=middle, end=end, index="2"),
+                    t_score,
+                    degrees_of_freedom,
```

Comment:
I assume these new statistics are the proposed replacements for the percentage.

Reply:
So these won't replace percentage, but create a scroe that lets us know how confident we can be for each trend